ethcore/res/ethereum/multisig_test.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
The file doesn't seem to be used anywhere. Would be really awesome to get a unit test for this.